app/search.py

```diff
@@ -494,6 +497,65 @@ def _query_fields(self, fields):  # pylint: disable=too-many-locals
                 f'{fields} "{preNonDigitAndPreDigit}"~5',
                 f'{fields} "{infNonDigitAndPreDigit}"'
             ]
+
+        # to improve the fuzzyness of the search result the search text has to be changed
+        # digit/text combinations are replaced with (digit|digit/text) p.e. 4a -> (4|4a)
+        # quorum operatar is used for query text fuzzy matching
+        if fuzzySearch:
+            # minimal length of single words that will be trimmed if only one word is in the query
+            # string
+            minLength = 5
+
+            def convert_if_digit(text):
+                if isdigit(text):
+                    digit = re.findall(r'^\d+', text)[0]
```

Comment:
the goal here is to add a more flexible and fuzzy aware way to treat house numbers like:
4a   -> (4|4a)
123-c -> (123|123-c)
15b -> (15|15b)

otherwise a request like this one won't hit any documents in the index:
`rest/services/ech/SearchServer?type=locations&origins=address&searchText=Chemin des Abeilles 4a 1304 Senarclens`